.gitignore

```diff
@@ -5,6 +5,7 @@
 /tmp
 /deploy
 /screenshots
+/src/examples-highlights
```

Comment:
Just to not bloat the PR. @riavalon why aren't we using the gulp plugin `gulp-highlight` ([here](https://github.com/johannestroeger/gulp-highlight)).

This would make the task way smaller and also easier to maintain.